frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineMetrics.tsx

```diff
@@ -41,131 +43,163 @@ const PipelineMetrics: React.FC<PipelineDetailsTabProps> = ({ obj, customData })
-                </Card>
-              </GridItem>
-              <GridItem xl2={5} xl={12} lg={12} md={12} sm={12}>
-                <Card>
-                  <CardHeader>
-                    <CardTitle>{t('pipelines-plugin~Number of PipelineRuns')}</CardTitle>
-                  </CardHeader>
-                  <CardBody>
-                    <PipelineRunCount
-                      interval={interval}
-                      timespan={timespan}
-                      pipeline={obj}
-                      loaded={loaded}
-                      onLoad={graphOnLoad}
-                      queryPrefix={queryPrefix}
-                      metricsLevel={metricsLevel}
-                    />
-                  </CardBody>
-                </Card>
-              </GridItem>
+          </Grid>
+        </StackItem>
+      </Stack>
+    );
+  }, [
+    graphOnLoad,
+    hasUpdatePermission,
+    interval,
+    latestPipelineRun,
+    loaded,
+    metricsLevel,
+    obj,
+    queryPrefix,
+    t,
+    timespan,
+  ]);
 
-              {metricsLevel === PipelineMetricsLevel.PIPELINERUN_TASKRUN_LEVEL && (
-                <>
-                  <GridItem xl2={7} xl={12} lg={12} md={12} sm={12}>
-                    <Card>
-                      <CardHeader>
-                        <CardTitle>{t('pipelines-plugin~PipelineRun Duration')}</CardTitle>
-                      </CardHeader>
-                      <CardBody>
-                        <PipelineRunDurationGraph
-                          interval={interval}
-                          pipeline={obj}
-                          timespan={timespan}
-                          loaded={loaded}
-                          onLoad={graphOnLoad}
-                          queryPrefix={queryPrefix}
-                          metricsLevel={metricsLevel}
-                        />
-                      </CardBody>
-                    </Card>
-                  </GridItem>
-                  <GridItem xl2={5} xl={12} lg={12} md={12} sm={12}>
-                    <Card>
-                      <CardHeader>
-                        <CardTitle>{t('pipelines-plugin~TaskRun Duration')}</CardTitle>
-                      </CardHeader>
-                      <CardBody>
-                        <PipelineRunTaskRunGraph
-                          interval={interval}
-                          timespan={timespan}
-                          pipeline={obj}
-                          loaded={loaded}
-                          onLoad={graphOnLoad}
-                          queryPrefix={queryPrefix}
-                          metricsLevel={metricsLevel}
-                        />
-                      </CardBody>
-                    </Card>
-                  </GridItem>
-                </>
-              )}
-            </Grid>
-          </StackItem>
-        </Stack>
-      )}
+  return (
+    <>
+      <Helmet>
+        <title>{t('pipelines-plugin~Pipeline metrics')}</title>
+      </Helmet>
+      {Content}
```

Comment:
Hi @vikram-raj ,

Can you remove the <Helmet> from this page. Since we made the user telemetry update, the pagetitle will be picked by generic module.

Reply:
@lokanandaprabhu I think we should keep the Helmet not for telemetry but for better user experience as the page title should be set for all the pages.

Reply:
Removed the Helmet. PTAL.